en_us/install_operations/source/installation/native_installation.rst

```diff
@@ -1,10 +1,10 @@
-.. _Open edX Native Installation:
+.. _Lilac Open edX Native Installation:
```

Comment:
@nedbat, yes, I figured you'd bring this up. :)

My question is: do we really need a page just for Lilac?  It's already generic enough to serve Koa.  And if anything significant changes in the future, it's going to be an Ubuntu version - and by that time, I figure we won't be supporting the Native installation at all.